cpu/dwh.py

```diff
@@ -41,9 +43,9 @@ def setUp(self):
         self.minthreads = self.params.get(
             'minthrd', default=(500 + cpu.online_cpus_count()))
         self.maxthreads = self.params.get('maxthrd', default=None)
-        self.iothreads = self.params.get('iothrd', default=self.minthreads/2)
+        self.iothreads = self.params.get('iothrd', default=old_div(self.minthreads,2))
```

Comment:
Please provide a space here `old_div(self.minthreads, 2)`

Reply:
@harish-24 noticed it, will take care in the main patch. In this RFC I would be expecting comments on the approach.